cmd/osctl/cmd/install_linux.go

```diff
@@ -71,7 +63,8 @@ var installCmd = &cobra.Command{
 			log.Fatal(err)
 		}
 
-		if err = install.Install(cmdline.String(), data); err != nil {
+		i := installer.NewInstaller(cmdline, data)
```

Comment:
The changes in this PR essentially boil down to making installations this simple.